src/openforms/formio/formatters/formio.py

```diff
@@ -167,3 +170,41 @@ def format(self, component: Component, value: str) -> str:
             src=value,
             alt=text,
         )
+
+
+class CustomerProfileFormatter(FormatterBase):
+    ADDRESS_TYPE_LABELS: Mapping[SupportedChannels, str] = {
```

Comment:
I think you can reuse `CommunicationChannels` TextChoices here (from `openforms/prefill/contrib/customer_interactions/api/serializers.py`). You can move it to formio module since it's more related to the "Profile" component